api/resolver/v1/resolver_api.proto

```diff
@@ -34,6 +34,23 @@ service ResolverAPI {
     };
     option (clutch.api.v1.action).type = READ;
   }
+  rpc AutoComplete(AutoCompleteRequest) returns (AutoCompleteResponse) {
```

Comment:
nit: autocomplete is one word, please change throughout
```suggestion
  rpc Autocomplete(AutocompleteRequest) returns (AutocompleteResponse) {
```

Reply:
I just realized this, going to change it back unless there is a way to make the linter happy / ignore this?
```
➜ clutch git:(resolver-autocomplete-api) make api-lint
tools/compile-protos.sh -c "/Users/mcutalo/gocode/src/github.com/lyft/clutch/api" -l
--- /Users/mcutalo/gocode/src/github.com/lyft/clutch/api/resolver/v1/resolver_api.proto
 v1beta1". See https://docs.buf.build/faq for more details.
37:20:RPC request type "AutocompleteRequest" should be named "AutoCompleteRequest" or "ResolverAPIAutoCompleteRequest".
37:50:RPC response type "AutocompleteResponse" should be named "AutoCompleteResponse" or "ResolverAPIAutoCompleteResponse".
make: *** [api-lint] Error 1
```

Reply:
it needs to match everywhere, check :37

Reply:
ooo i see nice